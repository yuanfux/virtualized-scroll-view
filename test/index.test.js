import React from 'react';
import { mount } from 'enzyme';
import VirtualizedScrollview from '../src';

describe('<VirtualizedScrollview />', () => {
  test('renders correctly', () => {
    const wrapper = mount(<VirtualizedScrollview />);
    expect(wrapper).toMatchSnapshot();
  });
});
