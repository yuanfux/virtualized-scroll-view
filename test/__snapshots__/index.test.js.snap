// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VirtualizedScrollview /> renders correctly 1`] = `
<VirtualizedScrollview
  text="Hello World!"
>
  <View
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        }
      }
    >
      <Text>
        <Text>
          Hello World!
        </Text>
      </Text>
    </View>
  </View>
</VirtualizedScrollview>
`;
